kustomize/move2kube/overlays/prod/values.properties

```diff
@@ -0,0 +1,12 @@
+# name of the secret holding the ssh keys that will be used by move2kube resources
+sshSecretName=sshkeys
+# name of the broker used by Knative eventing resources
+brokerName=default
```

Comment:
why not `ssh_secret_name` and `broker_name` instead?

Reply:
it was just a copy from the Helm values.yaml as-is
but I agree, will change it using a convention like `__word1_word2_..._wordN__`